parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4641,6 +4641,11 @@ SqlNode CreateProcedureStmt() :
         e = ConditionalStmt()
     |
         e = CursorStmt()
+    |
+        // This lookahead ensures parser choose
```

Comment:
nit: choose -> chooses

Reply:
Good catch. Fixed in https://github.com/googleinterns/calcite/pull/280/commits/f952df71de6a6e78a60de66d49c01ae89b92040e